### PR TITLE
codeship pro

### PR DESCRIPTION
**Pull request by jonstorer**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM paasmule/rbenv
+
+RUN mkdir /app
+ADD . /app
+WORKDIR /app
+
+RUN rbenv install $(cat .ruby-version)
```
Comment (sftwninja) [inline Dockerfile]: Sneaky

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+COPY Gemfile Gemfile.lock ./
+RUN gem install bundler && bundle install -j20
```
Comment (danlong005) [inline Dockerfile]: what is -j20?

Comment (jonstorer) [reply Dockerfile]:
```
       --jobs=[<number>]
              The maximum number of parallel download and install jobs. The default is 1.
```

Comment (danlong005) [reply Dockerfile]: oh... ok.

**Pull request closed without merging**